pymodbus/datastore/context.py

```diff
@@ -27,6 +31,43 @@ def decode(self, fx):
         """
         return self._fx_mapper[fx]
 
+    async def async_getValues(self, fc_as_hex: int, address: int, count: int = 1) -> list[int | bool | None] | ExceptionResponse:
+        """Get `count` values from datastore.
+
+        :param fc_as_hex: The function we are working with
+        :param address: The starting address
+        :param count: The number of values to retrieve
+        :returns: The requested values from a:a+c
+        """
+        return self.getValues(fc_as_hex, address, count)
+
+    async def async_setValues(self, fc_as_hex: int, address: int, values: list[int|bool]) -> None:
+        """Set the datastore with the supplied values.
+
+        :param fc_as_hex: The function we are working with
+        :param address: The starting address
+        :param values: The new values to be set
+        """
+        return self.setValues(fc_as_hex, address, values)
+
+    def getValues(self, fc_as_hex: int, address: int, count: int = 1) -> list[int | bool | None] | ExceptionResponse:  # pylint: disable=unused-argument
```

Comment:
Exception response is wrong.

Why do you need to add getValues/setValues here, it was not needed earlier?

Reply:
bit_read_messages.py and register_read_messages.py execute-calls seems to expect to receive ExceptionResponse from getValues.

And it was added here, as async_get/setValues refer to it in here.

Reply:
Well another problem from remote, ExceptionResponse is NOT valid in get/set values.